package.json

```diff
@@ -31,12 +31,14 @@
   "homepage": "https://github.com/skatejs/web-components#readme",
   "dependencies": {
     "document-register-element": "1.2.0",
-    "skatejs-named-slots": "2.0.4"
+    "@webcomponents/custom-elements": "https://github.com/webcomponents/custom-elements/archive/a0f55ecbb17eee97aab877cfda9ac3a16b3b14ac.tar.gz",
```

Comment:
These don't have hard releases yet - but are NPM compatible - so I've pinned them to specific commits.